assets/sip-124/implementation/implementation.ts

```diff
@@ -0,0 +1,24 @@
+import stableStringify from "fast-json-stable-stringify";
+import { sha256 } from "@noble/hashes/sha256";
+import { concatBytes } from "@metamask/utils";
+import { base64 } from "@scure/base";
+
+export type VFile = { path: string; contents: string };
```

Comment:
```suggestion
export type VFile = { path: string; contents: string | Uint8Array };
```

Reply:
Not used in the example, I'll leave it out to simplify the example